packages/next/build/index.ts

```diff
@@ -961,3 +965,27 @@ export default async function build(dir: string, conf = null): Promise<void> {
 
   await telemetry.flush()
 }
+
+function generateClientSsgManifest(
+  prerenderManifest: PrerenderManifest,
+  { buildId, distDir }: { buildId: string; distDir: string }
+) {
+  const ssgPages: Set<string> = new Set<string>([
+    ...Object.keys(prerenderManifest.routes),
+    ...Object.keys(prerenderManifest.dynamicRoutes),
+  ])
+
+  const clientSsgManifestPaths = [
+    '_ssgManifest.js',
+    '_ssgManifest.module.js',
```

Comment:
We only want to generate this if modern mode is enabled right?

Reply:
Done